### PR TITLE
fix lora from pretrained.

### PR DESCRIPTION
**Pull request by zzjjay**
### PR types

Bug fixes

### PR changes

Others

### Description

fix lora load.

### CODE REVIEW

File: paddlenlp/peft/lora/lora_model.py [hunk 1]
```diff
@@ -218,8 +218,9 @@ def _convert_tensor_parallel(self, lora_state_dict):
                 lora_name_action_mappings[v] = name_action_mappings[k]
 
         for name, action in lora_name_action_mappings.items():
-            tensor = lora_state_dict.pop(name)
-            lora_state_dict[name] = action(tensor)
+            if name in lora_state_dict:
+                tensor = lora_state_dict.pop(name)
+                lora_state_dict[name] = action(tensor)
```
Comment (lugimzzz) [inline paddlenlp/peft/lora/lora_model.py:223]: 添加一个else warning

File: paddlenlp/peft/lora/lora_model.py [hunk 2]
```diff
@@ -218,8 +218,11 @@ def _convert_tensor_parallel(self, lora_state_dict):
...
+            else:
+                logger.warning(f"{name} not found!")
```
Comment (lugimzzz) [inline paddlenlp/peft/lora/lora_model.py]: logger.warning(f"{name} not found in lora_state_dict! ")

File: paddlenlp/peft/lora/lora_config.py [hunk 3]
```diff
@@ -71,6 +71,7 @@ class LoRAConfig:
             "help": "The model multi head dimension.Only for LoRAMergedLinear and ColumnParallelLoRAMergedLinear."
         },
     )
+    do_qat: bool = field(default=False, metadata={"help": "Whether the lora model would do quant-aware training"})
```
Comment (lugimzzz) [inline paddlenlp/peft/lora/lora_config.py:74]: finetune_generation.py也要同步修改

File: paddlenlp/peft/lora/lora_model.py (hunk 2 above)
Comment (zzjjay) [reply paddlenlp/peft/lora/lora_model.py]: done

File: paddlenlp/peft/lora/lora_config.py (hunk 3 above)
Comment (zzjjay) [reply paddlenlp/peft/lora/lora_config.py:74]: done

**Review by lugimzzz — APPROVED** [review]
lgtm

**Pull request merged**